package-system/vulkan-validationlayers/Findvulkan-validationlayers.cmake

```
@@ -18,7 +18,7 @@ set(PATH_TO_DLL ${CMAKE_CURRENT_LIST_DIR}/vulkan-validationlayers/lib/release)
 set(_DLL_NAME ${PATH_TO_DLL}/${CMAKE_SHARED_LIBRARY_PREFIX}VkLayer_khronos_validation${CMAKE_SHARED_LIBRARY_SUFFIX})
 if (${PAL_PLATFORM_NAME} STREQUAL "Windows")
     set(${MY_NAME}_RUNTIME_JSON_DEPENDENCIES ${PATH_TO_DLL}/VkLayer_khronos_validation.json)
-elseif (${PAL_PLATFORM_NAME} STREQUAL "Android")
+else
```

These `else` needs parenthesis.
```suggestion
else()
```